<html><h1>Validation FAILED.</h1><p>Build Time: 25 minutes 58 seconds 979 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/cfc951544e1b0e7c1ccfc10c2a29946e0d249803">cfc951544e1b0e7c1ccfc10c2a29946e0d249803</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 1 formatting issues:
     [echo] 1: ./commerce-openapi-util/src/main/resources/com/liferay/commerce/openapi/util/config/rest-domain.config expected:&lt;...erator.enabled=false[]&gt; but was:&lt;...erator.enabled=false[
     [echo] ]&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:426)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:254)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [echo] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [echo] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/87//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>48 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/87//testReport">here</a>.</h5></div></html>

--- a/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/display/context/CommerceAccountDisplayContext.java
+++ b/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/display/context/CommerceAccountDisplayContext.java
@@ -278,6 +278,10 @@ public class CommerceAccountDisplayContext {
 			CommerceAccount commerceAccount, String actionId)
 		throws PortalException {
 
+		if (commerceAccount == null) {
+			return false;
+		}
+
 		return _modelResourcePermission.contains(
 			_commerceAccountRequestHelper.getPermissionChecker(),
 			commerceAccount, actionId);
@@ -295,8 +299,7 @@ public class CommerceAccountDisplayContext {
 	public boolean hasCommerceAccountModelPermissions(String actionId)
 		throws PortalException {
 
-		return _modelResourcePermission.contains(
-			_commerceAccountRequestHelper.getPermissionChecker(),
+		return hasCommerceAccountModelPermissions(
 			getCurrentCommerceAccount(), actionId);
 	}
 

--- a/commerce-account-web/src/main/resources/META-INF/resources/view_user.jsp
+++ b/commerce-account-web/src/main/resources/META-INF/resources/view_user.jsp
@@ -28,7 +28,7 @@ portletURL.setParameter("mvcRenderCommandName", "viewCommerceAccountUser");
 
 <portlet:renderURL var="editCommerceAccountURL">
 	<portlet:param name="mvcRenderCommandName" value="editCommerceAccountUser" />
-	<portlet:param name="commerceAccountId" value="<%= String.valueOf(commerceAccount.getCommerceAccountId()) %>" />
+	<portlet:param name="commerceAccountId" value="<%= (commerceAccount != null) ? String.valueOf(commerceAccount.getCommerceAccountId()) : StringPool.BLANK %>" />
 	<portlet:param name="userId" value="<%= String.valueOf(selectedUser.getUserId()) %>" />
 	<portlet:param name='<%= PortletQName.PUBLIC_RENDER_PARAMETER_NAMESPACE + "backURL" %>' value="<%= portletURL.toString() %>" />
 </portlet:renderURL>
@@ -57,72 +57,74 @@ portletURL.setParameter("mvcRenderCommandName", "viewCommerceAccountUser");
 	</section>
 </div>
 
-<div class="commerce-cta is-visible">
+<c:if test="<%= commerceAccount != null %>">
+	<div class="commerce-cta is-visible">
+		<c:if test="<%= (selectedUser.getUserId() != user.getUserId()) && commerceAccountDisplayContext.hasCommerceAccountModelPermissions(ActionKeys.UPDATE) %>">
+			<aui:button cssClass="commerce-button commerce-button--big js-invite-user" onClick='<%= renderResponse.getNamespace() + "openUserRolesModal();" %>' value="roles" />
+		</c:if>
+	</div>
+
+	<div class="commerce-account-container">
+		<commerce-ui:table
+			dataProviderKey="commerceAccountUserRoles"
+			filter="<%= commerceAccountDisplayContext.getAccountFilter() %>"
+			itemPerPage="<%= 5 %>"
+			namespace="<%= renderResponse.getNamespace() %>"
+			pageNumber="1"
+			portletURL="<%= portletURL %>"
+			tableName="commerceAccountUserRoles"
+		/>
+	</div>
+
+	<portlet:actionURL name="editCommerceAccountUser" var="editCommerceAccountUserURL" />
+
+	<aui:form action="<%= editCommerceAccountUserURL %>" method="post" name="editCommerceAccountUserFm">
+		<aui:input name="<%= Constants.CMD %>" type="hidden" value="EDIT_ROLES" />
+		<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
+		<aui:input name="commerceAccountId" type="hidden" value="<%= commerceAccount.getCommerceAccountId() %>" />
+		<aui:input name="userId" type="hidden" value="<%= selectedUser.getUserId() %>" />
+		<aui:input name="selectedRoleIds" type="hidden" />
+	</aui:form>
+
 	<c:if test="<%= (selectedUser.getUserId() != user.getUserId()) && commerceAccountDisplayContext.hasCommerceAccountModelPermissions(ActionKeys.UPDATE) %>">
-		<aui:button cssClass="commerce-button commerce-button--big js-invite-user" onClick='<%= renderResponse.getNamespace() + "openUserRolesModal();" %>' value="roles" />
+		<commerce-ui:user-roles-modal
+			commerceAccountId="<%= commerceAccount.getCommerceAccountId() %>"
+			componentId="userRolesModal"
+			userId="<%= selectedUser.getUserId() %>"
+		/>
+
+		<aui:script>
+
+			Liferay.provide(
+				window,
+				'<portlet:namespace />openUserRolesModal',
+				function(evt) {
+					const userRolesModal = Liferay.component('userRolesModal');
+					userRolesModal.open();
+				}
+			);
+
+			Liferay.componentReady('userRolesModal').then(
+				function(userRolesModal) {
+					userRolesModal.on(
+						'updateRoles',
+						function(selectedRoles) {
+							let selectedRoleIds = selectedRoles.map(
+								function(role) {
+									return role.id
+								}
+							).join(',');
+
+							document.querySelector('#<portlet:namespace />selectedRoleIds').value = selectedRoleIds;
+
+							userRolesModal.close();
+
+							submitForm(document.<portlet:namespace />editCommerceAccountUserFm);
+						}
+					);
+				}
+			);
+
+		</aui:script>
 	</c:if>
-</div>
-
-<div class="commerce-account-container">
-	<commerce-ui:table
-		dataProviderKey="commerceAccountUserRoles"
-		filter="<%= commerceAccountDisplayContext.getAccountFilter() %>"
-		itemPerPage="<%= 5 %>"
-		namespace="<%= renderResponse.getNamespace() %>"
-		pageNumber="1"
-		portletURL="<%= portletURL %>"
-		tableName="commerceAccountUserRoles"
-	/>
-</div>
-
-<portlet:actionURL name="editCommerceAccountUser" var="editCommerceAccountUserURL" />
-
-<aui:form action="<%= editCommerceAccountUserURL %>" method="post" name="editCommerceAccountUserFm">
-	<aui:input name="<%= Constants.CMD %>" type="hidden" value="EDIT_ROLES" />
-	<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
-	<aui:input name="commerceAccountId" type="hidden" value="<%= commerceAccount.getCommerceAccountId() %>" />
-	<aui:input name="userId" type="hidden" value="<%= selectedUser.getUserId() %>" />
-	<aui:input name="selectedRoleIds" type="hidden" />
-</aui:form>
-
-<c:if test="<%= (selectedUser.getUserId() != user.getUserId()) && commerceAccountDisplayContext.hasCommerceAccountModelPermissions(ActionKeys.UPDATE) %>">
-	<commerce-ui:user-roles-modal
-		commerceAccountId="<%= commerceAccount.getCommerceAccountId() %>"
-		componentId="userRolesModal"
-		userId="<%= selectedUser.getUserId() %>"
-	/>
-
-	<aui:script>
-
-		Liferay.provide(
-			window,
-			'<portlet:namespace />openUserRolesModal',
-			function(evt) {
-				const userRolesModal = Liferay.component('userRolesModal');
-				userRolesModal.open();
-			}
-		);
-
-		Liferay.componentReady('userRolesModal').then(
-			function(userRolesModal) {
-				userRolesModal.on(
-					'updateRoles',
-					function(selectedRoles) {
-						let selectedRoleIds = selectedRoles.map(
-							function(role) {
-								return role.id
-							}
-						).join(',');
-
-						document.querySelector('#<portlet:namespace />selectedRoleIds').value = selectedRoleIds;
-
-						userRolesModal.close();
-
-						submitForm(document.<portlet:namespace />editCommerceAccountUserFm);
-					}
-				);
-			}
-		);
-
-	</aui:script>
 </c:if>

